[fyi] How to reset your Android keystore (#137)
expo/fyi: android-reset-keystore.md

# Reseting your Android Upload key
## Background

> All newer apps (and many others) that are live on the Play Store are opted into [Google Play App Signing](https://support.google.com/googleplay/android-developer/answer/9842756?hl=en). With Play App Signing, Google maintains the signing key used to ultimately verify that your app is authentic. Meanwhile, when your app is built, it is signed with a separate key known as the "Upload key." Google Play checks your upload key when you go to publish your app to ensure the app binary is from you. If you lose this key, as long as you still have access to your Play Store account, you can reset it and continue publishing your app.

Reviewer: ```suggestion
[Google Play App Signing](https://support.google.com/googleplay/android-developer/answer/9842756?hl=en) is a security service where Google manages the app signing key on behalf of you as a developer.  All newer apps (and many others) that are live on the Play Store are opted into Play App Signing. 

With Play App Signing, Google maintains the signing key used to verify that your app is authentic. Meanwhile, when your app is built, it is signed with a separate "Upload key". Google Play checks your upload key when you are going to publish your app to ensure the app binary is from you. If you lose this key, as long as you still have access to your Play Store account, you can reset it and continue publishing your app.
```

Let's divide this so that it doesn't read like a wall of text and is skimmable. I also added one line at the start to briefly explain what Play Signing is so that when someone is ready for the doc, they don't have to look here and there when they come across "Play App Signing" terminology.

> ## Reseting your key with EAS-managed credentials

Reviewer: ```suggestion
## Reset your key with EAS-managed credentials
```

Let's use short form verbs to maintain the consistency of this doc.

> Google provides the most up-to-date information how to reset your key in their [Play Console Help section on Play App Signing](https://support.google.com/googleplay/android-developer/answer/9842756?visit_id=637973748658459850-3395295471&rd=1#reset), so be sure to consult that if you have any questions. However, if you use EAS to manage your Android app credentials, the process is generally a little easier, because the EAS CLI can create the new upload keystore for you.

Reviewer: ```suggestion
Google provides the most up-to-date information on how to reset your key in their [Play Console Help section on Play App Signing](https://support.google.com/googleplay/android-developer/answer/9842756?visit_id=637973748658459850-3395295471&rd=1#reset), so be sure to consult that if you have any questions. However, the process is generally a little easier if you use EAS to manage your Android app credentials because the EAS CLI can create the new upload keystore for you.
```

### Generate a new Upload keystore
1. Run `eas credentials`.
2. Choose "Android".
3. Choose the correct build profile, generally your "production" build profile.

> 4. Choose the "Keystore" option.

Reviewer: ```suggestion
4. Choose the "Keystore: Manage everything needed to build your project" option.
```

Let's the actual settings name from the CLI so that there is no confusion in future.

> 5. Chose the option to your keystore.

Reviewer: ```suggestion
5. Chose the option to your keystore.
```

I think this a duplicate of step 4.

Author: actually, was missing the "delete" in there!

> 6. Now choose the option to create a new keystore.

Reviewer: ```suggestion
6. Now choose the option "Set up a new keystore".
```

> 7. Finally, choose the option to download your new keystore. This will display fingerprint info for your keystore and download a **.jks** file.

Reviewer: ```suggestion
7. Finally, choose "Download existing keystore" to download the latest keystore you have created in the previous step. This will display fingerprint info for your keystore and download a **.jks** file at the root of your project directory.
```

### Reset your upload key on the Play Store

> These directions can change at Google's discretion, so always best to check [their latest instructions](https://support.google.com/googleplay/android-developer/answer/9842756?visit_id=637973748658459850-3395295471&rd=1#reset). As of November 2023, you can reset your upload key via an option under your app in the Play Store Console:

Reviewer: ```suggestion
These directions can change at Google's discretion, so it is always best to check [their latest instructions](https://support.google.com/googleplay/android-developer/answer/9842756?visit_id=637973748658459850-3395295471&rd=1#reset). As of November 2023, you can reset your upload key via an option under your app in the Play Store Console:
```
